.github/workflows/reusable-wait-block-override.yml

```diff
@@ -15,10 +15,14 @@ on:
       RUNS_ON:
         required: true
         type: string
-
+      CONTINUE:
```

Comment:
can we pick a different variable name for this? `CONTINUE` makes me think we're inside a loop and will be re-running the loop.

what if instead we called this `PERFORM_TEST` instead? Or alternatively `SKIP_TEST`, which would then require us to also adjust the values we pass in.

Reply:
will do!